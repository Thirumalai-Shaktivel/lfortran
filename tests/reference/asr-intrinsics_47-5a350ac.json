{
    "basename": "asr-intrinsics_47-5a350ac",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_47.f90",
    "infile_hash": "edc3de0432d45424733cd480e323511734eca302c80a93bfdc6c4a09",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_47-5a350ac.stdout",
    "stdout_hash": "7a4ff5af5199f33200c513cd35a332aed74fc3350ebdbf2765a7281c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}