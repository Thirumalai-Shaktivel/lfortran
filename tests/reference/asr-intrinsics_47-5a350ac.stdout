(TranslationUnit 
    (SymbolTable
        1
        {
            intrinsics_47:
                (Function 
                    (SymbolTable
                        2
                        {
                            kind:
                                (ExternalSymbol 
                                    2 
                                    kind 
                                    28 kind 
                                    lfortran_intrinsic_kind 
                                    [] 
                                    kind 
                                    Private
                                ), 
                            real:
                                (ExternalSymbol 
                                    2 
                                    real 
                                    4 real 
                                    lfortran_intrinsic_builtin 
                                    [] 
                                    real 
                                    Private
                                ), 
                            x:
                                (Variable 
                                    2 
                                    x 
                                    [] 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    intrinsics_47 
                    [] 
                    [] 
                    [(= 
                        (Var 2 x) 
                        (Cast 
                            (IntegerConstant 1 (Integer 4 [])) 
                            IntegerToReal 
                            (Real 8 []) 
                            (RealConstant 
                                1.000000 
                                (Real 8 [])
                            )
                        ) 
                        ()
                    )] 
                    () 
                    Source 
                    Public 
                    Implementation 
                    () 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    [] 
                    [] 
                    .false.
                ), 
            lfortran_intrinsic_builtin:
                (IntrinsicModule lfortran_intrinsic_builtin), 
            lfortran_intrinsic_kind:
                (IntrinsicModule lfortran_intrinsic_kind)
            
        }) 
    []
)
